split.py

```diff
@@ -1,4 +1,7 @@
-from sklearn.model_selection import train_test_split
+from skmultilearn.model_selection.iterative_stratification import iterative_train_test_split
```

Comment:
Is `skmultilearn` in `requirements.txt` ?

Reply:
That is what was missing !